app/javascript/components/common/documentTitle.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { useEffect } from 'react';
+
+const DocumentTitle: React.FC<{
+  title: string;
+}> = (props) => {
+  const {
+    title,
+    children,
+  } = props;
+  useEffect(() => {
+    document.title = title;
+  }, []);
+  return <>{children}</>;
```

Comment:
Is the `<> ... </>` necessary, or is the `children` fragment already of the right type?

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/issues/18051

Reply:
:cry: